packages/ckeditor5-bookmark/src/bookmarkui.ts

```diff
@@ -281,7 +281,7 @@ export default class BookmarkUI extends Plugin {
 			button.bind( 'isEnabled' ).to( updateBookmarkCommand );
 
 			this.listenTo( button, 'execute', () => {
-				this._showFormView();
+				this._showFormView( true );
```

Comment:
Im wondering about how to make it more descriptive. Cause this is not telling too much.

Maybe creta a descriptive variable, like:

```
const showBackButton = true;

this._showFormView( showBackButton );
```

Or maybe use object as a param of this/those functions?

```
_showFormView( { showBackButton: boolean } )
```

:thinking:

Reply:
Good point, imho, object approach is much more intuitive. I'll ref it this way.

Reply:
@pszczesniak done https://github.com/ckeditor/ckeditor5/pull/17562/commits/7b2541224a5060e753aecd276f2c96a1f3b67d8d